### ISSUE TITLE
Sorting by createdAt does not work

### ISSUE THREAD

**Issue by DominikGuzei** [mkatixlabs/tutorialMeteorjs#1]
Currently only the sorting after `total` seems to work correctly.

**Comment by mkatixlabs** [comment]
The sorting by createdAt it works, but in the db.invocies don't have any "record" with different day and same total.

**Comment by DominikGuzei** [comment]
Ok but from a UX stand-point i would expect that if i click on the createdAt sort button it becomes the main criteria to be sorted by (with total as secondary). Currently nothing happens when i click on it and that's not what i expect as a user.

**Comment by mkatixlabs** [comment]
yes you are right, but, what happens if are more sorted fields? then the "sorting button" has two behaviors, sort the collection and redefine the priority of all the other sorting fields. Maybe we need to think a solution where the user doesn't get lost trying to do the sorting and priorization.

**Comment by DominikGuzei** [comment]
Yeah, you're right there too – i guess it might be best option to allow only one sorting at once.
So the one you activate last is the only active one. This is how i would expect it as a user.
